src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -165,7 +165,10 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
                     Contract.ThrowIfNull(text);
 
                     var textSnapshot = text.FindCorrespondingEditorTextSnapshot();
-                    Contract.ThrowIfNull(textSnapshot);
+                    if (textSnapshot == null)
+                    {
+                        return;
```

Comment:
This should (minimally) be a `continue`, or otherwise we won't connect to the rest of the buffers.